### PR TITLE
Add Chinese translation ( utf-8 )

### PR DESCRIPTION
**Pull request by RonnyWu**
I think: "These translations are not perfect and need to be revised!"
Please give a test version, so that Chinese players try, 
I will be based on their suggestions to be amended!

### CODE REVIEW

File: Lang/zh-cn.cfg [hunk 1]
```diff
@@ -0,0 +1,986 @@
+// Auto generated by KSPDev Localization tool at: 9/2/2017 1:36:15 AM
+// Total strings: 231
+// Total words: 1188
+Localization
+{
+	en-us
```
Comment (ihsoft) [inline Lang/zh-cn.cfg:6]: This section should be named "zh-cn".

Comment (RonnyWu) [reply Lang/zh-cn.cfg:6]: Oh, yes, I forgot the changes after copying

Comment (RonnyWu) [reply Lang/zh-cn.cfg:6]: You can modify it after merging.

Comment (ihsoft) [reply Lang/zh-cn.cfg:6]: In general it's better to keep the pull requests atomic. But no worries, I'l merge and add a fix.

**Pull request merged**